corehq/apps/hqwebapp/static/hqwebapp/js/ui_elements/ui-element-input-map.js

```diff
@@ -83,8 +85,8 @@ hqDefine('hqwebapp/js/ui_elements/ui-element-input-map', [
                     });
 
                 }
-                mapKey = DOMPurify.sanitize(mapKey);
-                mapVal = DOMPurify.sanitize(mapVal);
+                mapKey = _.escape(DOMPurify.sanitize(mapKey));
+                mapVal = _.escape(DOMPurify.sanitize(mapVal));
```

Comment:
If we are escaping the strings, is there still a need to sanitize them?

For reference here's the original source of adding DOMPurify: https://github.com/dimagi/commcare-hq/pull/30695#discussion_r744919442